### DISCUSSION TITLE
Is there any community?

### DISCUSSION THREAD

**Discussion by killer4639** [Automattic/wp-calypso#55853] [General]
Hey everyone,
Is there a discord server or subreddit or something else to interact with the developers of the project?

**Comment by lynnbgriffith** [comment]
@killer4639 Thanks for reaching out! We don’t currently have a Discord or other community chat. If you’re looking to get involved with the broader WordPress ecosystem, there is a core Slack that is worth joining:
https://make.wordpress.org/chat/
If you’re interested particularly in Calypso, the best route is probably to create an issue with your question. Folks will route the issue or question to the relevant contributors or teams.

**Reply by eduarau** [reply]
Form and constant  where fyre
